modules/machine-config-overview.adoc

```diff
@@ -27,6 +27,12 @@ you can set up a custom pool of worker nodes that includes particular hardware
 features needed by an application. However, examples in this section focus
 on changes to the default pool types.
 
+[IMPORTANT]
+====
+A node can have multiple labels applied that indicate it is "master", "worker", etc,
+however it can only be a member of a *single* MachineConfigPool.
```

Comment:
```suggestion
however it can be a member of only a *single* machine config pool.
```
We're in the middle of changing the way that we [refer to API objects](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting), so since this isn't a specific reference, it should be split up.